test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -2549,6 +2549,14 @@ protected Vector<Float> vector(float[] v)
         return new Vector<>(v2);
     }
 
+    protected List<Float> vectorAsList(float[] v)
```

Comment:
Guava `Floats.asList()`?

Reply:
Perfect. Was looking for something like that...